ui/app/components/app/modals/disconnect-account/index.scss

```diff
@@ -0,0 +1,29 @@
+.disconnect-account-modal {
+  &__description {
+    color: #24292E;
+    margin-bottom: 24px;
+  }
+
+  &__cancel-button {
+    border: none;
+    margin-top: 12px;
+  }
+}
+
+.disconnect-account-modal-container {
+  .modal-container__header-text {
+    font-family: Roboto;
```

Comment:
The font specifications here can be turned into a variable or module or something. DRY

Reply:
f806d9df6